### ISSUE TITLE
Attributes not removed by freeze module

### ISSUE THREAD

**Issue by BowenBao** [pytorch/pytorch#39365]
## Description
There are still `SetAttr` and `GetAttr` remaining in the graph. Why is freeze module pass not removing them?

## To Reproduce

```python
class AttrMod(torch.nn.Module):
    def __init__(self, x):
        super().__init__()
        self.x = x

    def forward(self, x, y):
        self.x = x
        return self.x + y

x = torch.ones(1,2,3)
s_m = torch.jit.script(AttrMod(x))
print(s_m.graph)
s_m.eval()
f_s_m = torch._C._freeze_module(s_m._c)
f_s_m.dump()
"""
Scripted graph:
graph(%self : __torch__.AttrMod,
      %x.1 : Tensor,
      %y.1 : Tensor):
  %6 : int = prim::Constant[value=1]()
   = prim::SetAttr[name="x"](%self, %x.1)
  %4 : Tensor = prim::GetAttr[name="x"](%self)
  %7 : Tensor = aten::add(%4, %y.1, %6) # repro_ssa.py:29:15
  return (%7)

Frozen module graph:
module __torch__.___torch_mangle_0.AttrMod {
  parameters {
  }
  attributes {
    x = (1,.,.) =
      1  1  1
      1  1  1
    [ CPUFloatType{1,2,3} ]
  }
  methods {
    method forward {
      graph(%self : __torch__.___torch_mangle_0.AttrMod,
            %x.1 : Tensor,
            %y.1 : Tensor):
        %3 : int = prim::Constant[value=1]()
         = prim::SetAttr[name="x"](%self, %x.1)
        %4 : Tensor = prim::GetAttr[name="x"](%self)
        %5 : Tensor = aten::add(%4, %y.1, %3) # repro_ssa.py:29:15
        return (%5)

    }
  }
  submodules {
  }
}
"""

```

**Comment by bzinodev** [comment]
Folding SetAttr is not yet supported in Freezing. Typically they are seldom. Is there any interesting use case?

**Comment by BowenBao** [comment]
@bzinodev thanks for replying. One of such related case #37720, which originates from exporting `BatchNorm`.

**Comment by neginraoof** [comment]
This issue appears in test_word_language_model_GRU in test_pytorch_onnx_onnxruntime.

**Comment by janeyx99** [comment]
TorchScript has been deprecated in favor of torch.export as of 2.11. https://pytorch.org/blog/pytorch-2-11-release-blog/ Closing this issue as we are not further developing on it--please ping if you believe this issue was erroneously closed.
